spec/console_spec.rb

```diff
@@ -0,0 +1,104 @@
+require "console"
+require "stringio"
+
+RSpec.describe Console do
+  let(:input) { StringIO.new }
```

Comment:
All of this setup smells. My general rule is no more than two (maybe three) collaborators for any given class, I count at least 8 here.